dhis-2/dhis-services/dhis-service-acl/src/main/java/org/hisp/dhis/security/acl/DefaultAclService.java

```diff
@@ -541,7 +566,8 @@ public boolean canManage( User user, IdentifiableObject object )
         {
             if ( !canMakeExternal )
             {
-                errorReports.add( new ErrorReport( object.getClass(), ErrorCode.E3006, user.getUsername(), object.getClass() ) );
+                errorReports.add(
+                    new ErrorReport( object.getClass(), ErrorCode.E3006, user.getUsername(), object.getClass() ) );
```

Comment:
Could we take out some of the re-formatting? I know this is probably eclipse codestyle formatter but feels better to apply what has really changed in the source + gives a better overview.